src/app/service/datastore.service.ts

```diff
@@ -6,6 +6,7 @@ import { Payment} from "../models/payment";
 
 const config: DatastoreConfig = {
   baseUrl: 'https://kimchistan-api.herokuapp.com/api',
+  // baseUrl: '//localhost:3000/api/v1/',
```

Comment:
Is this url the right one?

Reply:
yes its the right address. we still need to have the localhost there since  no data has been added on the heroku site, and want to have the commented out localhost since we still need to manipulate the date according to the features we  work on